libraries/ESP8266WiFi/src/include/ClientContext.h

```diff
@@ -427,40 +449,47 @@ class ClientContext
             return false;
         }
 
-        size_t left = _datasource->available();
-        size_t can_send = tcp_sndbuf(_pcb);
-        if (_pcb->snd_queuelen >= TCP_SND_QUEUELEN) {
-            can_send = 0;
-        }
-        size_t will_send = (can_send < left) ? can_send : left;
         DEBUGV(":wr %d %d %d\r\n", will_send, left, _written);
```

Comment:
The debug lines fail to compile due to them trying to dump variables that were just removed (`left`)